docs/source/index.rst

```diff
@@ -53,6 +53,7 @@ Check out the :doc:`usage` section for further information, including :doc:`inst
    buffs
    cli
    command
+   data_sources
```

Comment:
This should not be a top level package, recommendation made to move in another comment.